Fix Examples layout page

jonataswalker opened the pull request:
Let's give a more solid approach to the Examples page.

This PR fixes #4888 and remove the `position:absolute` made by #4831.

Now there's a `div` tying up the Logo `a#href` and the Search `form` (or Version `select`) together.

ahocevar commented on examples/resources/common.js:
```diff
@@ -19,10 +19,10 @@
 
   if (window.location.host === 'localhost:3000') {
-    return;
+    // return;
```
Was this change meant to be in the pull request?

jonataswalker replied:
Surely not.

ahocevar commented:
@jonataswalker If you can address my comment, I can include this fix in the 3.14.2 release, since we have a few more fixed regressions already.

jonataswalker commented:
Addressed. Even after that [great explanation](https://github.com/openlayers/ol3/pull/4868#issuecomment-187785482) by Tim about rebasing and force pushing I'm still not sure:

I removed the unintended `// return;`, `git commit` and `git push --force origin branch-name`.

Is this all to do?

bartvde commented:
You should rebase and then force push

```
git rebase -i HEAD~2
git push origin -f [branch_name]
```

jonataswalker commented:
@bartvde thanks. Still in time for this one?

ahocevar commented:
Still in time. There is one more unmerged regression fix.

ahocevar commented:
But you still have two commits. When you do the rebase, change the word 'pick' in front of the 2nd commit to 'fixup'.

ahocevar commented:
Alternatively, you can do something else in this case:
```
git reset HEAD^
git add .
git commit --amend
```
followed by a force push.

jonataswalker commented:
I guess this is it, a single commit. :coffee:

ahocevar commented:
Awesome, thanks @jonataswalker
